avocado/plugins/list.py

```diff
@@ -276,31 +245,23 @@ def configure(self, parser):
     def run(self, config):
         verbose = config.get('core.verbose')
         write_to_json_file = config.get('list.write_to_json_file')
-        resolver = config.get('list.resolver')
-        runner = 'nrunner' if resolver else 'runner'
         config['run.ignore_missing_references'] = True
-        config['run.test_runner'] = runner
+        config['run.test_runner'] = 'nrunner'
         try:
-            if not resolver:
-                try:
-                    loader.loader.load_plugins(config)
-                    loader.loader.get_extra_listing()
-                except loader.LoaderError as error:
-                    LOG_UI.error(error)
-                    return exit_codes.AVOCADO_FAIL
+            loader.loader.load_plugins(config)
+            loader.loader.get_extra_listing()
```

Comment:
I think that we should also remove the loaders form list, because we are using the resolvers for listing. The loaders are used only in avocado-instrumented-runner.